routes/boycott/index.tsx

```diff
@@ -189,15 +206,22 @@ export const handler: Handler = {
             },
           ]
           : []),
+        // ...(ctx.state.country
+        //   ? [
+        //     {
+        //       $match: {
+        //         "attachedAlternatives.countries": ctx.state.country
+        //           .toLowerCase(),
+        //       },
+        //     },
+        //   ]
+        //   : []),
```

Comment:
Let's remove this comment if this code is obsolete at this point (we can get it back in case we needed it) unless you see otherwise of course